src/pds/aossrequestsigner/run.py

```diff
@@ -42,13 +43,14 @@ def run(
 
     auth = RequestsAWSV4SignerAuth(credentials, aws_region, "aoss")
 
+    request_f = requests.post if data else requests.get
     url = urllib.parse.urljoin(aoss_endpoint, request_path)
     if verbose:
-        print(f"Making request to url: {url}")
+        print(f"Making {'POST' if data else 'GET'} request to url: {url}")
```

Comment:
Is the output of this program ever the input to another (Unix philosophy)? If so, maybe add `file=sys.stderr` to `print()` so verbose diagnostics aren't parsed

Reply:
Not unforeseeable that someone would like to pipe it to `jq` - will fix

Reply:
ceca1f40